sphinx_favicon/__init__.py

```diff
@@ -113,31 +115,19 @@ def create_favicons_meta(pathto: Callable, favicons: FaviconsDef) -> Optional[st
     See Also:
         https://www.sphinx-doc.org/en/master/templating.html#path
     """
-    meta_favicons = ""
-
-    # generate meta for favicon dict
+    # force cast the favicon config as a list
     if isinstance(favicons, dict):
-        meta_favicons += generate_meta(_static_to_href(pathto, favicons))
-    # generate meta for list of favicon dicts
-    elif isinstance(favicons, list) and isinstance(favicons[0], dict):
-        for favicon in favicons:
-            meta_favicons += generate_meta(_static_to_href(pathto, favicon)) + "\n"
-    # generate meta for list of favicon URLs
-    elif isinstance(favicons, list):
-        for favicon in favicons:
-            if any(favicon.startswith(x) for x in ABSOLUTE_HREF_STARTERS):
-                attrs = {"href": favicon}
-            else:
-                attrs = _static_to_href(pathto, {FILE_FIELD: favicon})
-            meta_favicons += generate_meta(attrs) + "\n"
-    else:
-        logger.warning(
-            "Invalid config value for favicon extension."
-            "Custom favicons will notbe included in build."
-        )
-        return None
```

Comment:
Why shouldn't we keep a warning like this?

Reply:
I though it was an overkill, but if you think people will set up things like this: 
```python
favicons = [
    ["https://secure.example.com/favicon/favicon-16x16.png"],
    ["https://secure.example.com/favicon/favicon-32x32.png"],
    ["https://secure.example.com/favicon/apple-touch-icon-180x180.png"],
]
```

Then ok... I set it back in the last commit !

Reply:
Thanks! :)